operations/agent-flow-mixin/dashboards/prometheus.libsonnet

```diff
@@ -409,6 +409,9 @@ local remoteWritePanels(y_offset) = [
       dashboard.newMultiTemplateVariable('component', |||
         label_values(agent_wal_samples_appended_total{cluster="$cluster", namespace="$namespace", instance=~"$instance", component_id=~"prometheus\\.remote_write\\..*"}, component_id)
       |||),
+      dashboard.newMultiTemplateVariable('component_path', |||
+        label_values(agent_wal_samples_appended_total{cluster="$cluster", namespace="$namespace", instance=~"$instance", component_id=~"prometheus\\.remote_write\\..*", component_path=~".*"}, component_path)
+      |||),
```

Comment:
nit: I think the order of these should be swapped; the order here is most broad to most specific, so based on that path goes before component ID.